### PR TITLE
Gift Entry: Ensure instructions do not rely solely on sensory characteristics [A11y]

### PR DESCRIPTION
**Pull request by psadie-sfdo**
# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Review by CrystalPreston-Watson — APPROVED** [review]

**Review by ptewson-sfdo — COMMENTED** [review]
Looks good!

**Review by ptewson-sfdo — APPROVED** [review]

**Review by vmartinov-sforg — APPROVED** [review]
Looks good to me!

File: src/labels/CustomLabels.labels [hunk 1]
```diff
@@ -6039,7 +6039,7 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>Form Fields main section text block</shortDescription>
-        <value>Click a section, then select a field in the sidebar to add it to the section in the main screen. For each field, you can edit the field label, mark it as required, assign a default value or remove it from the section. Adjust the layout of the gift entry form by creating sections and reordering sections or fields.</value>
+        <value>Select a section, then select a field in the Form Fields &amp; Field Bundles sidebar to add it to this section in the main Configure Form Fields screen. For each field, you can edit the field label, mark it as required, assign a default value or remove it from the section. Adjust the layout of the gift entry form by creating sections and reordering sections or fields.</value>
```
Comment (melissabarber) [inline src/labels/CustomLabels.labels:6042]:
```suggestion
        <value>Select a section, then select a field in the Form Fields &amp; Field Bundles sidebar to add it to this section in the main Configure Form Fields screen. For each field, you can edit the field label, mark it as required, assign a default value or remove it from the section. Adjust the layout of the Gift Entry form by creating sections and reordering sections or fields.</value>
```

**Comment by melissabarber** [comment]
@psadie-sfdo & @ptewson-sfdo I added one small update to the label. I realized I hadn't capitalized Gift Entry previously. Other than that, I'm signed off on this PR.

**Comment by psadie-sfdo** [comment]
[Tracking W-7954185](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B0000008U7gJIAS/view)

**Review by ptewson-sfdo — APPROVED** [review]
Still looks good!

**Pull request merged**